questions-encodings.mjs

```diff
@@ -33,14 +18,27 @@ if (fs.existsSync(filename)) {
   console.log(`${prevNumQs} questions from in previous generated encodings.`)
 }
 
-const urlQuery =
-  'https://stampy.ai/w/api.php?action=ask&query=[[Canonically%20answered%20questions]]|format%3Dplainlist|%3FCanonicalQuestions&format=json'
-const r = await (await fetch(urlQuery)).json()
-const response = r.query.results['Canonically answered questions'].printouts.CanonicalQuestions
-console.log(`${response.length} questions fetched from the stampy.ai wiki.`)
+if (process.argv.length < 3) {
+  console.log("ERROR: Missing required argument ${CODA_TOKEN}",
+    "\nUSAGE: node questions-encodings.mjs ${CODA_TOKEN}")
+  exit(1)
+}
 
-const questions = response.map(({fulltext}) => fulltext)
+const CODA_TOKEN = process.argv[2]
+const CODA_DOC_ID = 'fau7sl2hmG'
+const table = 'Answers' 
+const params = 'useColumnNames=true&query=Status:%22Live%20on%20site%22&limit=1000' 
+const url =
+  `https://coda.io/apis/v1/docs/${CODA_DOC_ID}/tables/${table}/rows?${params}`
+
+const r = await (await fetch(url, {headers: {Authorization: `Bearer ${CODA_TOKEN}`}})).json()
```

Comment:
Very relieved I'm not the only person who does this `await (await fetch()).json()` :D